src/Compatibility/GenAPI/Microsoft.DotNet.GenAPI.Tool/Program.cs

```diff
@@ -97,13 +106,23 @@ static int Main(string[] args)
 
             rootCommand.SetAction((ParseResult parseResult) =>
             {
+                var excludeFiles = parseResult.GetValue(excludeApiFilesOption);
+                var includeFiles = parseResult.GetValue(includeApiFilesOption);
+                var hasExcludes = excludeFiles is not null && excludeFiles.Length > 0;
+                var hasIncludes = includeFiles is not null && includeFiles.Length > 0;
+                if (hasExcludes && hasIncludes)
+                {
+                    //TODO: what's the right way to handle this condition?
```

Comment:
The old tool applied exclusions after inclusions https://github.com/dotnet/arcade/blob/c80e079caaf7e892889651abd089f7569243f0f4/src/Microsoft.DotNet.GenAPI/GenAPITask.cs#L347